src/objects.js

```diff
@@ -1,41 +1,50 @@
 const createPerson = (name, age) => {
-  // your code here
+return {name, age};
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object.age;
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
-};
+  return person.age > 65
+  };
 
 const getAges = people => {
-  // your code here
+  return people.map(people => people.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(people => people.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+ return cars.filter(cars => cars.manufacturer === 'Honda');
 };
 
 const averageAge = people => {
-  // your code here
+ const totalAge = people.reduce((accAge, eachPerson) => {
+  return accAge + eachPerson.age;
+}, 0);
+  return totalAge / people.length;
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age,
+    introduce: introducePerson => {
+    return `Hi Fred, my name is ${name} and I am ${age}!`;
+   }
+  };  
```

Comment:
```suggestion
  return {
    name,
    age,
    introduce: introducePerson => {
      return `Hi Fred, my name is ${name} and I am ${age}!`;
   }
  };  
```
Indentation on the return line should be increased, and optionally you can use the shorthand JS for `name` and `age`